modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/controller/main/ProjectController.java

```diff
@@ -619,6 +619,16 @@ public void patch(
 		}
 	}
 
+	@GET
+	@Path("/{groupId}/usage/reset/preview")
+	@RolesAllowed(RoleConstants.SITE_ADMINISTRATOR)
+	public FaroSubscriptionDisplay previewResetProjectUsageDisplays(
```

Comment:
is this new endpoint for debugging purpose only?

Reply:
Yes. As we will need to reset monthlyValues, it might be good to see how the new values will look like :smile: